lambdas/accesstoken/src/main/resources/log4j2.yaml

```diff
@@ -0,0 +1,20 @@
+Configuration:
+  status: warn
+  appenders:
+    Lambda:
+      name: Lambda
+      JsonTemplateLayout:
+        eventTemplateUri: "classpath:JsonLayout.json"
+        eventTemplateAdditionalField:
+          - key: session-id
+            value: "$${ctx:sessionId:-unknown}"
+          - key: request-id
+            format: JSON
+            value: '{"$resolver": "pattern", "pattern": "%X{AWSRequestId}"}'
+  Loggers:
+    Root:
+      level: info
```

Comment:
Does this mean we only this INFO level logs in this format? Or will it work for WARN/ERROR levels as well?

Reply:
Yeah, it'll log from info up. If you set it to trace for example, you'd get all the logs. Set it to error and you'd just get error.